web/login.php

```diff
@@ -40,6 +40,27 @@
         exit;
     }
 
+    // Check for external authentication by web server
```

Comment:
please use spaces for indentation, not tabs. See `.editorconfig` in root. Can be fixed automatically with `yarn phpcs` (or `docker exec -it elabftw yarn phpcs`)